less/blocks.less

```diff
@@ -1,53 +1,62 @@
 
 .blocks {
+  // Display & Visibility
   display: block;
+
+  .block {
+    // Box Model
+    margin-top: 0.5rem;
+    margin-bottom: 0.75rem;
+    // Typography
+    color: var(--block-color);
+
+    &__header {
+      // Typography
+      font-weight: 600;
+      font-size: 2rem;
+      line-height: 1.5rem;
+    }
+
+    &__summary {
+      // Box Model
+      padding: 1rem 0 1.25rem;
+    }
+  }
 }
 
+// TODO: Does a top level media query generate any valid code? Does this do anything?
 @media (min-width: @bp-larger-than-phablet) {
   display: grid;
   grid-template-columns: repeat(2, minmax(0, 1fr));
   column-gap: 5rem;
   row-gap: 1rem;
 }
 
-.block {
```

Comment:
Looks like we're assuming that `.block` will only ever be used inside `.blocks`. That's fine with me as long as we've made sure that that's already true.

Reply:
In our current implementation a 'block' only ever appears within 'blocks'. So this is true for now, this just solidifies it as a requirement